OasisDEX/oasis-borrow
# Aave Open flow in 1 step (deposit + adjust) + simulation w/no wallet

# [Open flow in 1 step (deposit + adjust) in stead of 2](https://app.shortcut.com/oazo-apps/story/7220/open-flow-in-1-step-deposit-adjust-in-stead-of-2)

## Changes 👷‍♀️
- moved the risk adjustment to the first step, available immediately after inputting the amount
- moved the wallet requirement further down the flow so the user is able to simulate the position without wallet connection
  
## How to test 🧪
- go to any aave (earn or multiply) view with no wallet connection
- deposit amount and risk adjustment should be on one view
- you should be able to simulate the position
- you should be redirected to `/connect` when it's necessary
